base/logging.h

```diff
@@ -15,7 +15,270 @@
 #ifndef BASE_LOGGING_H_
 #define BASE_LOGGING_H_
 
-#include "glog/logging.h"
+#include <sstream>
+#include <limits>
+
+#include "base/macros.h"
+#include "base/types.h"
+
+namespace sling {
+
+#ifndef LOG
```

Comment:
Maybe add a similar #if-guard for the new base/flags.h, in case some piece of code #includes sling/base/flags.h and another flags.h?

Reply:
Good idea. I have added a guard on DEFINE_VARIABLE.